src/pyfaktory/client.py

```diff
@@ -128,7 +130,7 @@ def connect(self) -> bool:
         self.logger.info(f"Client lifecycle state is {self.state}")
 
         self.logger.info("Opening connection...")
-        if socket.has_dualstack_ipv6():
+        if socket.has_dualstack_ipv6() and self.enable_ipv6:
```

Comment:
Do we need the dualstack check anymore if we are flagging this as an option? Is it possible for an IPv6 connection to not support dualstack IPv6?